lib/IDE/CompletionLookup.cpp

```diff
@@ -1822,9 +1886,13 @@ void CompletionLookup::addMacroExpansion(const MacroDecl *MD,
     Builder.addRightParen();
   }
 
-  if (!MD->getResultInterfaceType()->isVoid()) {
+  auto roles = MD->getMacroRoles();
+  if (roles.containsOnly(MacroRole::Extension)) {
```

Comment:
Why do we need to handle extension macros specially here?

Reply:
It's a typo if Expression 😉

Reply:
Oh, expression makes sense